src/routes/consent.js

```diff
@@ -2,6 +2,10 @@ module.exports = {
   method: 'GET',
   path: '/consent',
   handler: function (request, reply) {
-    reply.view('consent')
+    let data = {
+      url: 'https://www.youtube.com/embed/bGIhghnseYk?rel=0&amp;autoplay=1&amp;controls=0&amp;showinfo=0',
```

Comment:
Thanks for the feedback. Good idea. We will change this at a later stage. At the moment I feel we are a bit behind and we have other things to focus on.